WalletWasabi/Tor/Socks5/Pool/TorHttpPool.cs

```diff
@@ -166,6 +166,16 @@ public async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, ICi
 						throw new HttpRequestException("Failed to handle the HTTP request via Tor.", e);
 					}
 				}
+				catch (TorConnectCommandFailedException e)
```

Comment:
Shouldn't this catch the exceptions from ObtainFreeConnectionAsync? If yes, then it should be outer by one scope.